fedora/python-flask-whooshee.spec

```diff
@@ -0,0 +1,64 @@
+%global mod_name flask-whooshee
+
+Name:           python-flask-whooshee
+Version:        0
+Release:        1%{?dist}
+Summary:        Whoosh integration
+
+License:        GPLv2+
+URL:            https://github.com/bkabrda/flask-whooshee
+Source0:        https://pypi.python.org/packages/source/f/%{mod_name}/%{mod_name}-%{version}.tar.gz
```

Comment:
Dumb question here, will Packit edit the specfile for itself and change the `Source0` line for what it generates in `create_archive` via

```
python3 setup.py sdist --dist-dir ./fedora/
bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
```

?

Or it will always use the PyPI tarball?

Reply:
Yes, packit overrides Source0 (try `packit srpm`).